assets/naf-process-chile-artesanales-schema.json

```diff
@@ -0,0 +1,56 @@
+[
```

Comment:
@smpiano we can gather all schemas into one since the attributes are the same. We can even merge it with the Panamanian one and have `naf-processed-schema.json`

Reply:
@enriquetuya if that is ok, I will gather all schemas, The `external_id` field will be added to the Panamanian schema to store the Chilean data coming from `XR` field.